src/utils/styles/partial/_breakpoints.scss

```diff
@@ -0,0 +1,4 @@
+$breakpoint-small: "only screen and (min-width: 481px)";
```

Comment:
use mobile first approach, what I have seen is you have used this to define classes (should be avoided)